packages/store-api/src/typeDefs/query.graphql

```diff
@@ -0,0 +1,72 @@
+type StoreProductEdge {
+  node: StoreProduct!
+  cursor: String!
+}
+
+type StoreProductConnection {
+  pageInfo: StorePageInfo!
+  edges: [StoreProductEdge!]!
+}
+
+
+type StoreCollectionEdge {
+  node: StoreCollection!
+  cursor: String!
+}
+
+type StoreCollectionConnection {
+  pageInfo: StorePageInfo!
+  edges: [StoreCollectionEdge!]!
+}
+
+enum StoreProductIDField {
+  id
+  slug
+}
+
+input StoreProductID {
+  field: StoreProductIDField!
+  value: ID!
+}
+
+enum StoreSort {
+  price_desc
+  price_asc
+  orders_desc
+  name_desc
+  name_asc
+  release_desc
+  discount_desc
+  score_desc
+}
+
+input StoreSelectedFacet {
+  key: String!
+  value: String!
+}
+
+enum StoreFacetType {
+  BOOLEAN
+  RANGE
+}
+
+type StoreSearchResult {
+  products: StoreProductConnection!
+  facets: [StoreFacet!]!
+}
+
+type Query {
+  product(locator: StoreProductID!): StoreProduct!
+
+  search(
+    first: Int!
+    after: String
```

Comment:
Could you add comments to these arguments? 🥺

Reply:
I think we do not need comments for that, once `first` and `after` are required for pagination.

Reply:
What about linking to the relay pagination spec?

Reply:
> I think we do not need comments for that, once first and after are required for pagination.

Yeah, I guessed they were hahahaha but if you don't know about the stile of pagination, these names don't mean much. Specially `after`. And comments would solve that.

> What about linking to the relay pagination spec? Something like:

@tlgimenes That's perfect!